### PR TITLE
docs: remove duplicate words

### PR DESCRIPTION
**Pull request by aanchal88**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by aanchal88** [comment]
Hi @jessicajaniuk, please merge the PR.

File: packages/animations/PACKAGE.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ route-based animations, and for programmatic animation controls that allow an en
 and reverse an animation sequence.
 
 @see Find out more in the [animations guide](guide/animations).
-@see See what polyfills you might need in the [browser support guide](guide/browser-support).
+@see What polyfills you might need in the [browser support guide](guide/browser-support).
```
Comment (jessicajaniuk) [inline packages/animations/PACKAGE.md:29]: This is not a duplicate word removal. Please revert this file.

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]
Thanks for helping improve the docs. Please address the feedback and I can re-review.

**Comment by aanchal88** [comment]
Hi @jessicajaniuk , I have re-opened another PR for the same. Please refer https://github.com/angular/angular/pull/53673.
Thanks

**Pull request closed without merging**